### DISCUSSION TITLE
Working spots and a new workbench suggestion.

### DISCUSSION THREAD

**Discussion by ozcodex** [jeremyshannon/Exile#153] [Ideas]
**TL;DR:** Nerfing the free working spots to the minimal to survive, and create new workbenchs that will replace the current working spots.

**P.S:** I've reading all that [DokimiCU](https://github.com/DokimiCU) wrote on issue https://github.com/jeremyshannon/Exile/issues/138 so my suggestion can be ignored. Even so I will leave it written here, maybe it can be useful for new ideas.

I see in your latest changes that new working spots have been added, and I see that as a great idea, love the grinding spot specially, but I miss the weaving frame and the hammering log.

### About the rules

Im aware of the rule of Function first:

> I’ve been reluctant to add anything that is purely decorative. Almost everything has some use. The use is why it got added to the game. I break this rule if aesthetics (or fun) requires it.

And maybe this suggestion can be seen as "decorative" but maybe can be something that goes good with the following rules:

- Clarity: crafting stations ideally have one page of crafts (In this case interpreted as a low amount of crafts)
- Secrets: This is a game to reward experimentation, creativity, and exploration. (Making the player figure out which tool will need)
- The Edge of Impossible: You should be forced to make difficult decisions without knowing in advance if they will work. ( creating a situation where can be wasting resources in a bench or tool that dont need yet)

### The suggestion

**leave the free spots with the minimal amount of recipes for basic survivalism**
Reduce the amount of available recipes on the crafting spot to basic tools and the next working bench, reduce the kind of seeds that can be extracted on the threshing spot, to only the groups of 6 and remove the woody plants (too hard to be removed by hand) and the palnts hard to find in the initial days of exile.

**create an intermediate tier of tools**
thin tools or working benchs will have the crafts that the spots currently have.
maybe bringing back the hammering block and the weaving frame (or similars if there are a reason to keep they removed), but also adding new ones like the "threshing sledge" (made with sticks and gravel)  or the working bench (just a short table to work more comfortable).

** the decorative and personal preference part **
This is just a thing of personal preferences, I whould love to be able to craft all the things in the late-game in benches and working stations without using the working spots, so I can have a nice house without the color spots in the floor (just for aesthetics), I hope to make myself understood.

**Comment by DokimiCU** [comment]
Regarding aesthetics you are right. That is one of the trade-offs of the crafting spot approach.

As suggested, this could be solved by having built equivalents to all crafting spots (where appropriate). e.g. a "General Workbench" that has all the craft spot crafts. That's perfectly acceptable time to put priority on aesthetics. Doesn't harm game balance.

To give an advantage to making the bench perhaps they could have better resource efficiency in the recipe. It is easier to work at a bench than in a muddy puddle!
